### ISSUE TITLE
Disabling Credit Cards

### ISSUE THREAD

**Issue by samcheneytlc** [Pikaju/FlutterBraintree#178]
Is there a way to disable credit cards specifically using the drop in UI and also remove the "Credit or" text from the sheet?

<img width="407" alt="Screenshot 2024-11-21 at 4 29 23 PM" src="https://github.com/user-attachments/assets/7b21bdaf-f866-4958-b20a-358ba01f623c">

**Comment by BunnyBuddy** [comment]
Yes there is, but you may need to change it from the native code.
